lib/commands/kendoui-notes.ts

```diff
@@ -0,0 +1,68 @@
+///<reference path="../.d.ts"/>
+"use strict";
+import {KendoUIBaseCommand} from "./kendoui-base";
+
+class KendoUINotesCommand extends KendoUIBaseCommand implements ICommand {
+
+	constructor(private $logger: ILogger,
+		private $opener: IOpener,
+		private $prompter: IPrompter,
+		$errors: IErrors,
+		$kendoUIService: IKendoUIService,
+		$loginManager: ILoginManager,
+		$options: IOptions,
+		$project: Project.IProject) {
+
+		super($errors, $project, $kendoUIService, $loginManager, $options);
+	}
+
+	allowedParameters: ICommandParameter[] = [];
+
+	canExecute(args: string[]): IFuture<boolean> {
+		return ((): boolean => {
+			let number = args ? parseInt(args[0]) : true;
+			if (!number) {
+				this.$errors.failWithoutHelp(`The argument ${args[0]} is not applicable to this command`);
+			}
+
+			if(args.length > 1) {
+				this.$errors.fail("This command only accepts one parameter.");
+			}
+
+			if(this.$options.core && this.$options.professional) {
+				this.$errors.fail("You cannot specify core and professional options simultaneously.");
+			}
+
+			return true;
+		}).future<boolean>()();
+	}
+
+	execute(args: string[]): IFuture<void> {
+		return (() => {
+			let packages = _.filter(this.getKendoPackages().wait(), p => p.HasReleaseNotes);
+			if (packages.length === 1) {
+				this.$opener.open(_.first(packages).ReleaseNotesUrl);
+				return;
+			}
+
+			this.$logger.out("You can review release notes for the following Kendo UI packages.");
+			this.$logger.out(this.getKendoPackagesAsTable(packages));
+			let schema: IPromptSchema = {
+				type: "input",
+				name: "packageIdx",
+				message: "Enter the index of the package",
+				validate: (value: string) => {
+					let num = parseInt(value, 10);
+					return !isNaN(num) && num >= 1 && num <= packages.length ? true : "Valid values are between 1 and " + packages.length;
```

Comment:
If packages' length happens to be equal to 1, that package's release notes are immediately shown and the command's execution is stopped